tests/41end-to-end-keys/08-cross-signing.pl

```diff
@@ -682,7 +682,17 @@
       })->then( sub {
          sync_until_user_in_device_list( $user1, $user2 );
       })->then( sub {
-         matrix_get_e2e_keys( $user1, $user2_id );
+         repeat_until_true {
```

Comment:
generally, I see `repeat_until_true` as an antipattern, because it means that genuine test failures get turned into test timeouts.

Prefer `retry_until_success`.